content/en/apps/guides/hosting/monitoring/setup.md

```diff
@@ -29,6 +29,48 @@ The solution provides both an overview dashboard as well as a detail dashboard.
 
 ### Setup
 
+```mermaid
```

Comment:
this diagram is complex and also in need of a number of fixes.  For example, this area is illegible because of overlapping text and lines: 

![image](https://user-images.githubusercontent.com/8253488/235499549-d5b746e6-197c-420a-9f0c-6480104e06be.png)

let's break adding this diagram out into it's own PR so as to not block hugo uplift and adding mermaid feature

Reply:
Good call. I will open a separate PR for this.